bahmni-emr/package/helm/values.yaml

```diff
@@ -1,8 +1,12 @@
+global:
+  storageClass: ""
+
 replicaCount: 1
 
 metadata:
   labels:
     environment: dev
+    name : openmrs-document-images-pvc
```

Comment:
I think this can be removed. This has been moved to `claimName` I hope.

Reply:
yes, will remove it